k8s/k8s.go

```diff
@@ -1982,8 +1982,10 @@ func (k *k8sOps) CreatePod(pod *v1.Pod) (*v1.Pod, error) {
 	return k.client.Core().Pods(pod.Namespace).Create(pod)
 }
 
-func (k *k8sOps) GetPods(namespace string) (*v1.PodList, error) {
-	return k.getPodsWithListOptions(namespace, meta_v1.ListOptions{})
+func (k *k8sOps) GetPods(namespace string, labelSelector map[string]string) (*v1.PodList, error) {
+	return k.getPodsWithListOptions(namespace, meta_v1.ListOptions{
+		LabelSelector: mapToCSV(labelSelector),
```

Comment:
What happens to list options when the map is empty/empty list of label selectors?